docs/architecture.md

```diff
@@ -50,12 +49,12 @@ a generic Linux distribution.
 
 Elemental includes an operator that is responsible for managing OS upgrades
 and managing a secure device inventory to assist with zero touch provisioning.
-See the full operator docs at [Elemental-operator](https://github.com/rancher-sandbox/Elemental-operator/blob/main/README.md)
+See the project at [elemental-operator](https://github.com/rancher/elemental-operator/#readme)
 
-## Elemental Teal
+## The underlaying OS
 
-Elemental Teal is based off of SUSE Linux Enterprise (SLE) Micro for Rancher.  There is no specific dependency on
+Elemental is based off of SUSE Linux Enterprise (SLE) Micro for Rancher.  There is no specific dependency on
```

Comment:
This is not fully correct
- Elemental is the project/stack, (elemental, -toolkit, -operator, -cli, ...) which targets the Rancher stack - this is the "blue" part
- Elemental Teal is the combination of Elemental ("blue") with SLE Micro for Rancher ("green")

Reply:
what, why would you bring blue into the picture? Blue refers in toolkit to Fedora....way to make it more confusing :laughing: